ext/storage_sources/gcp_store/gcp_storage_source.cpp

```diff
@@ -466,50 +520,66 @@ static int
 gcp_object_list(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *directory,
   const char *prefix, char ***object_list, uint32_t *count)
 {
-    WT_UNUSED(file_system);
-    WT_UNUSED(session);
-    WT_UNUSED(directory);
-    WT_UNUSED(prefix);
-    WT_UNUSED(object_list);
-    WT_UNUSED(count);
-
-    return 0;
+    return gcp_object_list_helper(
+      file_system, session, directory, prefix, object_list, count, false);
 }
 
+// Allocate and initialize an array of C strings from vector of strings.
+//
+// Requires count <= objects.list().
```

Comment:
I think you mean `objects.size()`

Reply:
Fixed in [2247e06](https://github.com/wiredtiger/wiredtiger/pull/8794/commits/2247e061d343aef915a73a5310503cefd4c82e0e)